README.md

```diff
@@ -4,7 +4,28 @@ Virtual Coffee's Task Challenges are created/aggregated by VC members to help ot
 
 ## Benefits
 
-These task challenges will greatly benefit the entire Virtual Coffee community. They will serve our members who are current working professionals by allowing them to up-skill in select technologies and tools. They will also help our job seekers, especially those with no prior professional experience in the tech industry. The unique added value of the task challenges is that they're modeled after real-world scenarios, lending these exercises a quality that far surpasses what's readily available on sites like LeetCode and Codewars.
+These task challenges will greatly benefit the entire Virtual Coffee community by providing members with opportunities to up-skill in select technologies and tools through practical, real-world scenarios.
+
+### For Task Challenge Posters
+
+**Share Expertise:** Contribute your knowledge and experience to help others grow.
+**Community Contribution:** Play a pivotal role in fostering a learning environment.
+**Mentorship:** Guide and mentor fellow members through their learning journey.
+
+### For Task Challenge Takers
+
+**Practical Experience:** Engage with challenges modeled after real-world scenarios, providing applicable experience.
+**Skill Development:** Build and enhance your skill set with hands-on tasks.
+**Portfolio Building:** Create a portfolio of practical projects to showcase your abilities to potential employers.
+**Feedback and Improvement:** Receive constructive feedback from experienced professionals to refine your skills.
```

Comment:
**suggestion:** I would add one more bullet point here...
**issue:** Need to add bullets so they don't render funny

```suggestion
### For Task Challenge Takers

- **Practical Experience:** Engage with challenges modeled after real-world scenarios, providing applicable experience.
- **Skill Development:** Build and enhance your skill set with hands-on tasks.
- **Team Building:** Review peers' code and give targeted, helpful feedback.
- **Portfolio Expansion:** Create a portfolio of practical projects to showcase your abilities to potential employers.
- **Feedback and Improvement:** Receive constructive feedback from experienced professionals to refine your skills.
```